examples/pytorch/wan/pipeline_wan_flashinfer.py

```
@@ -0,0 +1,247 @@
+            "suffix on 'single' (e.g. 'single-fa3') is forwarded to "
+            "single_prefill_with_kv_cache."
+        ),
+    )
+    parser.add_argument("--skip-softmax-sparse", action="store_true")
+    parser.add_argument("--skip-softmax-threshold", type=float, default=1.0)
+    parser.add_argument("--prepare-weights", action="store_true")
+    parser.add_argument(
+        "--prompt",
+        default="Two anthropomorphic cats in comfy boxing gear and bright gloves fight intensely on a spotlighted stage.",
+    )
+    parser.add_argument("--negative-prompt", default="色调艳丽，过曝，静态，细节模糊不清，字幕，风格，作品，画作，画面，静止，整体发灰，最差质量，低质量，JPEG压缩残留，丑陋的，残缺的，多余的手指，画得不好的手部，画得不好的脸部，畸形的，毁容的，形态畸形的肢体，手指融合，静止不动的画面，杂乱的背景，三条腿，背景人很多，倒着走")
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)